simulator/infra/log.cpp

```diff
@@ -22,3 +22,7 @@ OStreamWrapper::~OStreamWrapper()
 {
     ostream.rdbuf( buffer);
 }
+
+
+std::ofstream* visualizer_logger = new std::ofstream(".\\logs.json", std::ofstream::out);
```

Comment:
For testing, these variables should be non-static and reside in "Root" class. Objects can access them recursively.
Additionally, file name should be configurable from command line.